Map/src/main/java/dk/sdu/se/f23/InVasion/map/MapPlugin.java

```diff
@@ -21,20 +20,16 @@
 import java.awt.image.BufferedImage;
 import java.io.IOException;
 import java.io.InputStream;
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.Map;
-import java.util.NoSuchElementException;
+import java.util.*;
 
 
-public class MapPlugin /*implements PluginService*/ {
+public class MapPlugin{
```

Comment:
please add space between MapPlugin and { 

(jep, jeg er træls, men du gjorde det samme mod mig @Darchie4 )

Reply:
Fixed